test/Generics/pack-shape-requirements.swift

```diff
@@ -59,3 +59,9 @@ struct Ts<T...> {
     }
   }
 }
+
+// CHECK-LABEL: expandedParameters(_:transform:)
+// CHECK-NEXT: Generic signature: <T..., Result... where T.shape == Result.shape>
+func expandedParameters<T..., Result...>(_ t: T..., transform: ((T) -> Result)...) -> (Result...) {
```

Comment:
Can you add a third generic parameter and change the outer return type, to ensure that we don't equate the shapes of all three generic parameters?